third_party/packages/split/lib/split.dart

```diff
@@ -40,11 +42,22 @@ class _SplitOptions {
   external List<num> get sizes;
 
   external List<num> get minSize;
+
+  /// minSize is ignored unless this property is set.
```

Comment:
How about "Whether to use the [minSize] property."

https://dart.dev/guides/language/effective-dart/documentation#prefer-starting-variable-getter-or-setter-comments-with-noun-phrases

Reply:
done.